client/state/data-layer/wpcom/concierge/schedules/appointments/index.js

```diff
@@ -39,6 +39,24 @@ export const bookConciergeAppointment = ( { dispatch }, action ) => {
 	);
 };
 
+export const rescheduleConciergeAppointment = ( { dispatch }, action ) => {
+	dispatch( updateConciergeBookingStatus( CONCIERGE_STATUS_BOOKING ) );
+
+	dispatch(
+		http(
+			{
+				method: 'POST',
+				path: `/concierge/schedules/${ action.scheduleId }/appointments/${ action.appointmentId }/reschedule`,
+				apiNamespace: 'wpcom/v2',
+				body: {
+					begin_timestamp: action.beginTimestamp / 1000,
```

Comment:
Maybe just leave a comment on this line similar to the one above in `toApi`: `// convert to UNIX timestamp.`

Reply:
Right, updated.